picard/util/webbrowser2.py

```diff
@@ -17,12 +17,14 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
+from sys import version_info
 import webbrowser
 
-from PyQt5 import QtWidgets
-
+from picard import log
 from picard.const import PICARD_URLS
 
+from PyQt5 import QtWidgets
```

Comment:
There's no reason to move this import here, it was in correct place.

Reply:
This is what `isort` did to the file.

Reply:
Yep, see #993. We are trying to sort this out :)